website/content/docs/upgrading/upgrade-to-1.10.x.mdx

```diff
@@ -31,3 +31,19 @@ Vault storage to an Etcd v3 cluster prior to upgrading to Vault 1.10.
 All storage migrations should have
 [backups](/docs/concepts/storage#backing-up-vault-s-persisted-data)
 taken prior to migration.
+
+ ### OTP Generation Process
```

Comment:
For this section, maybe rewrite, something along these lines:

Customers passing in OTPs during the the process of generating root tokens must modify the OTP generation to include an additional 2 characters before upgrading so that the OTP can be xor-ed with the encoded root token. This change was implemented as a result of the change in the prefix from hvs. to s. for service tokens.